### ISSUE TITLE
Remote Audio Vu-meter

### ISSUE THREAD

**Issue by PurpleHorrorRus** [Palakis/obs-websocket#198]
##### Issue type
Feature request

##### Description
It would be nice if you could monitor, for example, the level of the microphone when I speak. If this already exists or such a request has already been, sorry for this and my bad English.

**Comment by Palakis** [comment]
Depends on #275

**Comment by tt2468** [comment]
Duplicate of #152
